quiz/quiz.go

```diff
@@ -139,10 +134,11 @@ func (q *Quiz) Start(maxQuizTime int) error {
 // timeQuiz waits on the maxQuizTime and sends a "timeout" string down the channel when it is expired.
 // takeAndTimeQuiz returns the first thing down the channel, so the maxQuizTime will 'interrupt' the quiz if
 // it's still going
-func timeQuiz(quizTimer *time.Timer, c chan string) {
-	<-quizTimer.C
-	fmt.Println("\nTimer has run out!")
-	c <- "timeout"
+func timeQuiz(maxQuizTime int, c chan string) {
+	fmt.Printf("Starting timer for %v seconds.\n", maxQuizTime)
+	time.AfterFunc(30 * time.Second, func() {
+		fmt.Println("\nTimer has run out!"); c <- "timeout"
```

Comment:
Go convention is that you should use more lines rather than longer lines (i.e. don't use semicolons except in if/switch/for statements)